resources/views/application/partials/_form_application.blade.php

```diff
@@ -16,7 +16,7 @@
   {{-- Test Type --}}
   <div class="field-group -mono {{ setInvalidClass('test_type', $errors) }}">
     {!! Form::label('test_type', 'Test Type:') !!}
-    {!! Form::select('test_type', array('PSAT' => 'PSAT', 'SAT' => 'SAT', 'PLAN' => 'PLAN', 'ACT' => 'ACT', 'Prefer not to submit scores' => 'Prefer not to submit scores')); !!}
+    {!! Form::select('test_type', array('PSAT' => 'PSAT', 'SAT (Out of 1600)' => 'SAT (Out of 1600)', 'SAT (Out of 2400)' => 'SAT (Out of 2400)', 'PLAN' => 'PLAN', 'ACT' => 'ACT', 'Prefer not to submit scores' => 'Prefer not to submit scores')); !!}
```

Comment:
Totally subjective, but one thing you could do is make the actual values you send in the form request different than the readable labels in the drop down so it is nicer to validate. I.E the user sees "SAT (Out of 1600)" but we send, store, and validate "SAT1600". I like validating against simple strings because I worry that I could introduce errors if you capitalize something incorrectly or add an extra space.

Reply:
@sbsmith86 I like the idea of that. I'd have to make a helper to render the test type so that we still always display `SAT (Out of 1600)` when the applicant reviews their application and when admins look at it. Right now it's just able to pull the exact value from the database. I'm making a not to come back and do that once the other work is finished.